ssl/config.h

```diff
@@ -45,7 +45,7 @@
 #define CONFIG_SSL_HAS_PEM 1
 #undef CONFIG_SSL_USE_PKCS12
 #define CONFIG_SSL_EXPIRY_TIME 24
-#define CONFIG_X509_MAX_CA_CERTS 150
+#define CONFIG_X509_MAX_CA_CERTS 10
```

Comment:
No need to create static array of 150 certs chain. ["Practically encountered chains have length 2 to 4 certificates (i.e. one root, three intermediate CA at most, and then the end-entity certificate)"](http://security.stackexchange.com/questions/38949/understanding-ssl-certificate-signing) , 10 for safety